dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/DefaultEventDataQueryService.java

```diff
@@ -378,7 +378,7 @@ public EventQueryParams getFromAnalyticalObject( EventAnalyticalObject object )
         for ( DimensionalObject dimension : ListUtils.union( object.getColumns(), object.getRows() ) )
         {
             DimensionalObject dimObj = dataQueryService.getDimension( dimension.getDimension(),
-                getDimensionalItemIds( dimension.getItems() ), date, null, true, idScheme );
+                getDimensionalItemIds( dimension.getItems() ), date, null, true, null, idScheme );
```

Comment:
Would be good to somehow avoid adding many null arguments, makes the code more error prone and harder to read.

Reply:
Yes, I agree, we are planning to redesign the way how we are passing parameters down to the methods.